### ISSUE TITLE
Import job with null value character causes import of list values to be emptied of data.

### ISSUE THREAD

**Issue by exeqtwasnotfree** [atrocore/atrocore#864]
Hello
When I have a character for example '--' or '$' as an empty value flag, the import of lists does not work.
For example I have a list field for "Status in ERP", this gets cleared if I use characters for empty value.

I need to use blank value characters if I don't want to import all attributes for all classifications and products.
A classification for a product type can contain 10 attributes, but my import file can contain 50 attributes, then I don't want to get a bunch of empty attributes that have nothing to do with the article.

**Comment by AnnZast** [comment]
Hello, 
- if you import character set as empty value, attribute is added to the product with an empty value, if the value was previously set, it also becomes empty
- if you import character set as Null value, the value of the attribute does not change if it was previously set. If the attribute has not been added to the product before, it is added
- if you want attribute not to be linked to the product, you should use "Marking for no relation" character. In this case, the attribute will not be added to the product, and if its value is already set for a certain product, it will be removed

**Comment by exeqtwasnotfree** [comment]
Thank you AnnZast for the info.
I have made more tries and tests, and I can see the impact of the respective settings.
It seems that in my environment, I can´t have one import for several classifications.
Ether I get many empty attributes on product that not suppose to have that attribute, (when using "null value").
Or several of the article based entities of type list gets cleared, (when using "empty value").
Or empty attributes marked for no relation, disappear on products that don´t have this value yet, but needs the attribute empty for now until value is entered manual later.

So my solution for now is to have one import job per classification and use empty value.
In my eyes the list-based field should not be cleared when using empty value, because i supply the value I want in the file.

**Comment by AnnZast** [comment]
Yes you're right. We plan to create another value for the import feed that will allow the attribute value to be skipped on import, regardless of whether it is set or not

**Comment by exeqtwasnotfree** [comment]
Yes, that sounds like a great idea.
Hope you find the time to implement this in the near future.
Thank you for the answer.
